js/modules/k6/ws/ws_test.go

```diff
@@ -1288,3 +1292,46 @@ func TestCookieJar(t *testing.T) {
 
 	assertSessionMetricsEmitted(t, metrics.GetBufferedSamples(ts.samples), "", sr("WSBIN_URL/ws-echo-someheader"), statusProtocolSwitch, "")
 }
+
+func TestWSConnectWithThrowErrorOption(t *testing.T) {
+	t.Parallel()
+
+	logHook := &testutils.SimpleLogrusHook{HookedLevels: []logrus.Level{logrus.WarnLevel}}
+	testLog := logrus.New()
+	testLog.AddHook(logHook)
+	testLog.SetOutput(io.Discard)
```

Comment:
I would argue you can move this in the `ThrowDisabled` as well. And arguably just splie this 1 test with 2 sub tests in two tests without any subTests :thinking:

Reply:
how about add two tests:
- TestWSConnectEnableThrowErrorOption: maybe not need ?
- TestWSConnectDisableThrowErrorOption

Reply:
Sounds good. I think we can keep both and have them next to each other as contrasting tests.

You might also want to check that there is no log in the case where we throw :thinking: